project_scrum/views/project_project.xml

```diff
@@ -29,11 +28,38 @@
             </field>
         </field>
     </record>
- 
+
+    <record id="act_project_tasks" model="ir.actions.act_window">
+        <field name="name">Tasks</field>
+        <field name="res_model">project.task</field>
+        <field name="view_mode">kanban,tree,form,calendar,pivot,graph</field>
+        <field name="context">{
+            'group_by': 'stage_id',
+            'search_default_project_id': [active_id],
+            'default_project_id': active_id,
+        }</field>
+        <field name="search_view_id" ref="project.view_task_search_form"/>
+        <field name="help" type="html">
+            <p class="oe_view_nocontent_create">
+                Here, you can create new tasks
+            </p>
+            <p>
+                You can now manage your tasks in order to get things done efficiently. Track progress, discuss, attach documents, etc.
+            </p>
+        </field>
+        <field
+            name="view_ids"
+            eval="[(5, 0, 0),
```

Comment:
no need to do the whole delete/create thing.
just do (6,0, [list of ids of the views you want])
view_ids is a M2M to model ir.actions.act_window. For readability sake, create view_ps_sprint_task_tree2  and view_ps_sprint_task_form2 above this stanza with their own XML and then refer to them in here as:

eval"[(6,0,[ ref('action_view_ps_sprint_task_tree2').id, ref('action_view_ps_sprint_task_form2').id])

Reply:
> view_ids is a M2M to model ir.actions.act_window

Actually I am not sure that is true as I see here https://github.com/OCA/OCB/blob/10.0/odoo/addons/base/ir/ir_actions.py#L309

And that's why I have to create those records on the fly there.

Reply:
still don't create them on the fly, create `act_window.view` records instead - this will create new records on every update